timeseries/tests/test_SimulatedTimeSeries.py

```diff
@@ -1,66 +1,142 @@
-from SimulatedTimeSeries import SimulatedTimeSeries
+""" test code for SimulatedTimeSeries
+This module contains tests for SimulatedTimeSeries.
+
+Methods
+-------
+make_simple_data(stop=None):
+    A function to generate simulated data
+make_stat_data(m, stop=None):
+    A function to generate normally distributed data with mean of 1e4 and standard deviation m
+test_type():
+    A function to test if input is a generator
+test_str():
+    A function to test __str__
+test_repr():
+    A function to test __repr__
+test_produce():
+    A function to test produce using data size of 50 and 5 different chunk sizes
+test_mean_std():
+    A function to test online_mean and online_std using data size 1e5 and chunk size 9e4
+    Error in last mean is limited to 0.1%, error in last standard deviation is limiated to 1%.
+
+"""
+
+
 from itertools import count
 from random import normalvariate, random
+from pytest import raises
+from SimulatedTimeSeries import SimulatedTimeSeries
 import numpy as np
 
+
 def make_simple_data(stop=None):
+    """
+    A function to generate simulated data
+
+    """
     for value in count():
         if stop and value > stop:
             break
         yield value
 
-def make_stat_data(m, stop=None):
+
+def make_stat_data(in_std, stop=None):
+    """
+    A function to generate normally distributed data with mean of 1e4 and standard deviation m
```

Comment:
"with ... standard deviation m", but it seems that the std here is in_std, not m?